webextensions/api/devtools.json

```
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools",
           "support": {
             "chrome": {
-              "version_added": "≤58"
+              "version_added": "3"
```

:x: https://github.com/chromium/chromium/commit/89ebc7edb982ebf5f8b185aa6e3aae6904eed965 first appeared in [4.0.212.1](https://github.com/chromium/chromium/releases/tag/4.0.212.1) before 4.[0.223].

```suggestion
              "version_added": "4"
```